api/src/usersAvatar/controllers/usersAvatar-controller.ts

```diff
@@ -14,13 +15,26 @@ import {
     ApiTags,
     ApiUnauthorizedResponse
 } from "@nestjs/swagger";
+import { createReadStream } from "fs";
 
 @ApiBearerAuth("defaultBearerAuth")
 @ApiTags("users-avatar")
 @Controller("users-avatar")
 class UsersAvatarController {
     constructor(private readonly usersAvatarService: UsersAvatarService) {}
 
+    @ApiUnauthorizedResponse()
+    @ApiOkResponse()
+    @ApiNotFoundResponse()
+    @Get(":id")
+    async getFile(@Param("id") id: number): Promise<StreamableFile> {
```

Comment:
If You get this by ID You should check authorization. Because in this way every user could get all of the files which 
which may not be what you want. In every previous version there was geting file by uuid and this is more complicated string than 1,2,3,4... id. So checking accesses wasn't required. On the other hand, information about the user's avatar is to some extent public information. So it's up to You.

1. Getting by sourcePath as Param
2. Leave it as it is

Reply:
#46 

Look at this, i know about it, but i don't have this code because it was in #43 i must wait to merge this, for have ep. User decorator.

Reply:
done